doc/whats_new/v1.1.rst

```diff
@@ -13,7 +13,11 @@ Changelog
 ---------
 
 - |Fix| A default HTML representation is shown for meta-estimators with invalid
-  parameters. :pr:`24015` by `Thomas Fan`_.
+  parameters. :pr:`24015` by `Thomas Fan`
```

Comment:
I think this turns off the linking for my name? 😅

Reply:
Oops, this is a multi-tasking typo. My bad! :facepalm: